dipy/viz/ui.py

```diff
@@ -3024,12 +3023,12 @@ def _setup(self):
 
         # Add up and down buttons
         arrow_up = read_viz_icons(fname="arrow-up.png")
-        self.up_button = Button2D([("up", arrow_up)])
+        self.up_button = Button2D(("up", arrow_up))
```

Comment:
I think the error comes from here. `Button2D([("up", arrow_up)])` is the right version. Can you confirm @dmreagan ?